packages/tasit-action/package.json

```diff
@@ -35,8 +35,10 @@
     "@babel/plugin-proposal-class-properties": "^7.2.3",
     "@babel/preset-env": "^7.1.6",
     "@babel/register": "^7.0.0",
+    "bn.js": "^4.11.8",
```

Comment:
We can remove this in a later PR, I guess, if we're going to use the ethers.js one.

Reply:
Sure, thanks to remind me :+1: